src/components/search/SearchFilters.jsx

```diff
@@ -19,74 +20,72 @@ export const FREE_ALL_TITLE = 'Free / All';
 
 const SearchFilters = () => {
   const size = useWindowSize();
-  const { showAllCatalogs, setShowAllCatalogs } = useContext(SearchContext);
+  const { refinementsFromQueryParams } = useContext(SearchContext);
   const showMobileMenu = useMemo(
     () => size.width < breakpoints.small.maxWidth,
     [size],
   );
-
   const freeAllItems = useMemo(() => [
     {
       label: 'Free to me',
-      value: !showAllCatalogs,
+      // eslint-disable-next-line no-bitwise
+      value: refinementsFromQueryParams[SHOW_ALL_NAME] ^ 1,
```

Comment:
Even though this goes against our eslint, I'm OK with using bitwise here. But maybe a comment explaining what it's doing would be helpful for our future selves?

Reply:
Yeah, good call.